### ISSUE TITLE
FIFA 07 Game Crash

### ISSUE THREAD

**Issue by Maarcis12** [hrydgard/ppsspp#19177]
### Game or games this happens in

ULES00440 - FIFA 07

### What area of the game / PPSSPP

Every time i play a match, no matter the mode, it just crashes with a blue screen of Bad Execution address

https://github.com/hrydgard/ppsspp/assets/26842473/e0828ec0-50f1-4812-b610-ee17af44c7f5

### What should happen

It shouldn't crash.

### Platform

Windows

### Mobile device model or graphics card (GPU)

Nvidia GeForce RTX 3050

### PPSSPP version affected

v1.17.1-545

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by anr2me** [comment]
Have you checked the CRC to see whether the ISO is good or not?

**Comment by Maarcis12** [comment]
yes, in fact, it was the first thing it when it bluescreened and it passed with flying colors.

**Comment by anr2me** [comment]
Could you find out which version was the last working version? if you knew that it works fine before.

**Comment by Maarcis12** [comment]
> Could you find out which version was the last working version? if you knew that it worked fine before.

Ok, after 2 hours of testing old builds, i found it, the last working build for the game was v1.17.1-264, after that, build 268 and onwards, game just bluescreens mid match.

**Comment by hrydgard** [comment]
Thanks, that means https://github.com/hrydgard/ppsspp/pull/19053 . I'll look into it soon, definitely before 1.18.

**Comment by hrydgard** [comment]
Yeah, this is Atrac streaming related. It breaks exactly in commit a1ac79c18354074d090824f44da612a74cdc7266 , between the two revisions mentioned. I'll try to fix it tomorrow.

**Comment by Maarcis12** [comment]
Ok, thank you so much for the response.

**Comment by Maarcis12** [comment]
And the blue screen sound crash is back.

**Comment by hrydgard** [comment]
Really? Hm.

**Comment by Maarcis12** [comment]
all the latest dev builds, thankfully for you, i've tested earlier builds to pinpoint the suspect build, it was build 602.

**Comment by Maarcis12** [comment]
Also, apologies for my super late response, my native Baltics are going through the power switchover, so i was prepping for that the whole day.

**Comment by Maarcis12** [comment]
any news on what could be the cause?

**Comment by ppmeis** [comment]
Testing latest Atrac3+ PR build, still issues:

![Image](https://github.com/user-attachments/assets/c571f768-c785-48db-a09c-8eeb6cb63153)
